recipes/qcustomplot/all/conanfile.py

```diff
@@ -45,9 +46,9 @@ def configure(self):
 
     def requirements(self):
         if Version(self.version) >= "2.0.0":
-            self.requires("qt/6.4.1")
+            self.requires("qt/6.6.0", transitive_headers=True)
         else:
-            self.requires("qt/5.15.7")
+            self.requires("qt/5.15.11", transitive_headers=True)
```

Comment:
```suggestion
            self.requires("qt/5.15.11", transitive_headers=True, transitive_libs=True)
```
Qt probably isn't used as header-only here, so transitive_libs is likely necessary.